continuum/scenarios/base.py

```diff
@@ -111,7 +123,7 @@ def _select_data_by_task(self, task_index: Union[int, slice]):
             # like PermutedMNIST or RotatedMNIST.
             selected_y = self.cl_dataset.class_remapping(selected_y)
 
-        return selected_x, selected_y, selected_t
+        return selected_x, selected_y, selected_t, task_index
```

Comment:
je comprends pas ce task_index en plus du selected_t, quelque commentaires en plus serait bienvenu :)

Reply:
I need it to discernate between multiple tasks. In Seg, a same image can have several tasks.

Yes I'll do doc/commentary :)